setup.py

```diff
@@ -238,7 +238,7 @@ def isgood(name):
     'package_data'      :  {'': ['*.sh', '*.json', 'VERSION', 'SDIST']},
 
     #'install_requires'  :  ['radical.pilot', 'pika', 'pandas', 'numpy', 'matplotlib'],
-    'install_requires'  :  ['radical.utils', 'pika', 'radical.pilot',
+    'install_requires'  :  ['radical.utils', 'pika==0.13.1', 'radical.pilot',
                             'pytest','hypothesis','sphinx'],
```

Comment:
What happens in other pika versions?

Reply:
They changed the order of the arguments in their latest release. We have a ticket for it: https://github.com/radical-cybertools/radical.entk/issues/331

Reply:
Ah, right - thanks.